### PR TITLE
Dropdown Styling

### PR DESCRIPTION
**Pull request by jakedcolbert**

### CODE REVIEW

File: src/lib/components/Navbar.svelte [hunk 1]
```diff
@@ -111,6 +112,7 @@ The navbar component.
                              */
                             props: { options: books.map((b) => b.bookCode) /*bookAbbreviations*/ }
                         },
+                        // 
                         Chapter: {
```
Comment (jakedcolbert) [inline src/lib/components/Navbar.svelte:116]: Chapter and Verse tabs should be removed here. They do not match the implementation in the Android Application.

File: src/lib/components/SelectGrid.svelte [hunk 2]
```diff
@@ -4,29 +4,65 @@ A component to display menu options in a grid.
...
     import { createEventDispatcher } from 'svelte';
+    import { globalConfig } from '$lib/data/stores';
     export let options = [''];
-    export let cols = 5;
+    export let cols = 6;
+
     const dispatch = createEventDispatcher();
     $: rows = Math.ceil(options.length / cols);
 
+    let textColor = $globalConfig.themes
```
Comment (jakedcolbert) [inline src/lib/components/SelectGrid.svelte:14]: Lots of lines just to get styling for the dropdown.

Comment (jakedcolbert) [reply src/lib/components/SelectGrid.svelte]: Looks better, but could benefit from a themeing store or themeing defined elsewhere

File: src/lib/components/SelectGrid.svelte [hunk 3]
```diff
@@ -4,10 +4,31 @@ A component to display menu options in a grid.
...
     import { createEventDispatcher } from 'svelte';
+    import { globalConfig } from '$lib/data/stores';
     export let options = [''];
-    export let cols = 5;
+    export let cols = 6;
+
     const dispatch = createEventDispatcher();
     $: rows = Math.ceil(options.length / cols);
+    let colors = (type: string, key: string) =>
+        $globalConfig.themes.find((x) => x.name === 'Normal').colorSets.find((x) => x.type === type)
```
Comment (chrisvire) [inline src/lib/components/SelectGrid.svelte:14]: 'Normal' should be change to a theme store ($theme).

File: src/lib/components/TabsMenu.svelte [hunk 4]
```diff
@@ -4,12 +4,16 @@ A component to display tabbed menus.
...
+    import { globalConfig } from '$lib/data/stores';
 
     export let options: App.TabMenuOptions = { '': { component: '', props: {} } };
     export let active = '';
-    export let _class = 'bg-primary';
     const dispatch = createEventDispatcher();
 
+    $: primaryColor = $globalConfig.themes
+        .find((x) => x.name === 'Normal') // TODO: change to fetch the current theme
+        .colorSets.find((x) => x.type === 'main').colors['PrimaryColor'];
```
Comment (chrisvire) [inline src/lib/components/TabsMenu.svelte:15]: 'PrimaryColor' should be changed to a style lookup.  Contact @davidmoore1 to find out the style name.

**Review by chrisvire — APPROVED** [review]
We can approve for now. But you should create an issue to clean up the hardcoding afte rwe have a theme store.

**Pull request merged**